contracts/Mailbox.sol

```diff
@@ -0,0 +1,30 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+
+contract Mailbox is Context {
+  mapping(address => string) private mailboxes;
+
+  event RegisterUser(address sender, string mailboxStorage);
+  
+  constructor() {
+    // what should we do on deploy?
```

Comment:
I'm thinking maybe we can have a string as constructor param for the `pendingEmailInboxAddress`? Or maybe we just have a function that will be able to update that value later on.

Definitely need a `pendingEmailInboxAddress: String` variable, as well as a setter (owner only) and getter.

Reply:
Ok I have implemented the private variable` _pendingInbox` and added the getter: `.pendingInbox()` and the setter `.updatePendingInbox(string memory inboxAddress)`